iron-icon.html

```diff
@@ -183,13 +183,20 @@
               this._iconset.removeIcon(this);
             }
           } else if (this._iconsetName && this._meta) {
-            this._iconset = /** @type {?Polymer.Iconset} */ (
-              this._meta.byKey(this._iconsetName));
-            if (this._iconset) {
-              this._iconset.applyIcon(this, this._iconName, this.theme);
-              this.unlisten(window, 'iron-iconset-added', '_updateIcon');
-            } else {
-              this.listen(window, 'iron-iconset-added', '_updateIcon');
+            // Since _updateIcon can be called multiple times, make sure we
+            // avoid cloning the SVG if the icon & theme haven't changed
+            if (this._iconName !== this._lastIconName ||
```

Comment:
Can we introduce two constants right above for these inequality comparisons to increase readability? I’d suggest `iconNameChanged` and `themeChanged`